### ISSUE TITLE
Not showing up in Fedora 38

### ISSUE THREAD

**Issue by rago666** [GaZaTu/im-emoji-picker#11]
I have properly setup fcitx5. Then I ran `wget -q https://raw.githubusercontent.com/GaZaTu/im-emoji-picker/master/install.sh && sh install.sh -f fcitx5 -r tags/nightly-build` and it successfully installed. But I can't find it in fcitx5. I also did a reboot but it didn't work either.

![Screenshot_20230728_141046](https://github.com/GaZaTu/im-emoji-picker/assets/51720144/c2e926a2-f3ab-4673-951a-b9df188e7405)

**Comment by GaZaTu** [comment]
@rago666 I don't remember what the KDE settings for fcitx5 look like but as a hint:
im-emoji-picker is not an input method when using fcitx5. It is a module running next to your default input method.
Modules are configurable in that dialog too but not under the input methods.
To just see if it works at all you can try pressing the default hotkey for the emoji picker: `Control+Alt+period`

**Comment by rago666** [comment]
Nope, doesn't work. But I did find a section called modules under configure addons but im-emoji-picker is not mentioned there.
![Screenshot_20230728_163635](https://github.com/GaZaTu/im-emoji-picker/assets/51720144/60658304-0363-4beb-97e0-4ad91213b50c)

**Comment by GaZaTu** [comment]
weird. i'll look into this when i'm home 😅

**Comment by GaZaTu** [comment]
you can execute `sudo cp /usr/lib/fcitx5/fcitx5imemojipicker.so /usr/lib64/fcitx5/` in your terminal for now. that should fix the issue.
i have to figure out how to handle this situation. on arch there is no lib64 or rather it's just a symlink to lib so it always worked for me as is. idk if i should just hardcode lib64 into the install script.

**Comment by rago666** [comment]
Thank you for your time. The command indeed make I'm Emoji Picker to show up in the module section. And the app work flawlessly under x11. 
But under wayland session, the app dies as soon as it launches. Running `sudo journalctl -p 3 -xb` shows this error during the failed launch
`fcitx5[2603]: Failed to create wl_display (No such file or directory)`
 `fcitx5[2603]: This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
`

**Comment by GaZaTu** [comment]
ok the only thing i can imagine at this point is that there must be some kind of bug with KDE (wayland) + fcitx5 because fcitx immediately sends the `deactivate` event after the window opens (which is obviously wrong) and then it goes dark.
if i start the fcitx daemon with `QT_QPA_PLATFORM=xcb` it kinda works but that's obviously no solution.

**Comment by milas** [comment]
I grabbed the logs from `fcitx5 --verbose '*=5'` and opened an issue with fcitx5:
 * fcitx/fcitx5#887
